lib/chai.js

```diff
@@ -4,14 +4,13 @@
  * MIT Licensed
  */
 
-var used = []
-  , exports = module.exports = {};
+var used = [];
 
 /*!
  * Chai version
  */
 
-exports.version = '3.5.0';
+exports.version = require('../package').version;
```

Comment:
@meeber @vieiralucas don't worry about reverting, our build process will write over this change when we release 4.0

@shvaikalesh We've already got the tooling that writes the version. We can definitely look at Rollup though; do you want to open an issue to discuss it?